src/render/drm_format_set.zig

```diff
@@ -0,0 +1,44 @@
+const std = @import("std");
+const c = @cImport(@cInclude("wlr/render/drm_format_set.h"));
+
+pub const DrmFormat = extern struct {
+    // Use comptime introspection to get the alignment of arch dependent field
+    format: u32 align(fieldAlignment(c.wlr_drm_format, "format")),
```

Comment:
I believe we could use `align(std.meta.fieldInfo(c.wlr_drm_format, .format).alignment)` here instead. It does the same thing as your `fieldAlignment()` function but leverages the `std` instead.